src/assets/svg-components/MassaToken.tsx

```diff
@@ -0,0 +1,46 @@
+export interface MAssaTokenProps {
+  theme?: string;
+}
+
+export function MassaToken(props: MAssaTokenProps) {
+  return (
+    <svg
+      width="24"
+      height="24"
+      viewBox="0 0 24 24"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <circle
+        cx="12"
+        cy="12"
+        r="12"
+        fill={props.theme === "theme-light" ? "#FFFFFF" : "#151A26"}
```

Comment:
Colors are already specified in the codebase. We don't need to repeat it here. Specially in raw. There's a logic behind Tailwind that maybe we are missing. We don't need to care about the theme as long as we use the named colors: primary, secondary etc. 

Please, check/sync with Lucas (dev) to understand better how this works.

Reply:
ok will do ! :)

Reply:
Solution has been discussed and implemented in most recent commit